BraveWallet/Panels/WalletPanelView.swift

```diff
@@ -33,7 +33,19 @@ public struct WalletPanelContainerView: View {
     case unlock
   }
 
-  @State private var visibleScreen: VisibleScreen = .loading
+  private var visibleScreen: VisibleScreen {
```

Comment:
just want make srue you dont want this to be a @State property anymore? i guess you dont need it anymore cuz you are listening the change for `keyringStore.keyring`.

Reply:
Yeah, it was only made to `@State` so I could preserve the animation when locking the wallet, but this caused some other issues, ex. when `WalletStore` leaks we can get stuck in loading state because the `keyringStore.keyring` never changes, so the `onChange(of:)` doesn't fire.